### DISCUSSION TITLE
How to debug "Uninitialized string offset 0" warnings

### DISCUSSION THREAD

**Discussion by tree28** [mpdf/mpdf#2059] [Q&A]
Hello,

I am a new Mpdf user. Really appreciating it but I have encountered the warning: 

`Uninitialized string offset 0 in ... vendor/mpdf/mpdf/src/Mpdf.php on line 16724`

I would like to know if anyone has advice for eliminating the warning. What typically causes it, what should I look for, is there any particular type of html or css that triggers the error that I should be aware of?

Many thanks in advance.

**Comment by finwe** [comment]
Most likely it is non-well-formed html or css. But to assess this better, create an issue with a simple reproducible example causing this, so the warning can be potentially fixed in the Mpdf code.

**Reply by tree28** [reply]
Thanks will do.
